### PR TITLE
Feat: New Sanity quick start and updated resources

### PR DESCRIPTION
**Pull request by kmelve**
#### What kind of changes does this PR include?

- New or updated content
- Changes to the docs site code

#### Description

Here is an overhauled new Quick Start for Sanity as well as updated resources.

### CODE REVIEW

**Comment by sarah11918** [comment]
Thanks so much for this contribution @kmelve !

Heads up that it, uh, came at a super unfortunate time.  😂  Between Launch week and our intensive planning sprint, our docs team isn't prioritizing new content right now, and CMS guides are the larger pieces of content we have to deal with from an editing, formatting, spinning up new projects and verifying standpoint. So, they always take a bit of work on a good day... and we haven't even really had a "good day" yet!

I notice you've also included a lot more links to Sanity + Astro content at the bottom, and that's amazing! Those can be approved right away without any special attention from our docs team required. So if you'd like to make a separate PR that only adds links, to get your content added and available in docs right away, please do so!

If you're OK to wait a little longer until we're back up to full-capacity and full-speed, then it's no problem to just wait until this PR merges. I'd *like* to say it will be a next week task, but do be prepared that our quarterly planning is our first priority for another week yet. We try to sneak in urgent fixes and simple merges when we can, but we're not committing to having full attention on new content until the following week.

Just to keep you up to speed! Thanks for your patience!

**Comment by kmelve** [comment]
You have our full understanding! Appreciate the update – let me know if there is anything I can do to make the review process easier for you 🙌

**Comment by kmelve** [comment]
Hi @sarah11918 – I just wondered if there was any timeline on this?

**Comment by sarah11918** [comment]
Hi @kmelve ! Our plan is to have these (we have a handful of CMS guides PRd all at the same time) completed by the end of next week! Thank you again for your patience, and we're looking forward to getting this in the docs!

File: src/content/docs/en/guides/cms/sanity.mdx [hunk 1]
```diff
@@ -2,21 +2,216 @@
...
+  output: 'hybrid',
+  integrations: [sanity({
+    projectId: '<your-project-id>',
+    dataset: '<dataset-name>',
+    useCdn: false, // See note on using the CDN
+    apiVersion: "2023-03-20", // insert the current date to access the latest version of the API
+    studioPath: '/admin' // If you want to access the Studio on a route
+  }), react()]
+});
+```
```
Comment (ElianCodes) [inline src/content/docs/en/guides/cms/sanity.mdx:113]:
``````suggestion
```javascript
// astro.config.mjs
import { defineConfig } from "astro/config";
import sanity from "@sanity/astro";
import react from "@astrojs/react";

export default defineConfig({
  output: 'hybrid',
  integrations: [sanity({
    projectId: '<your-project-id>',
    dataset: '<dataset-name>',
    useCdn: false, // See note on using the CDN
    apiVersion: "2023-03-20", // insert the current date to access the latest version of the API
    studioPath: '/admin' // If you want to access the Studio on a route
  }), react()]
});
```
``````

File: src/content/docs/en/guides/cms/sanity.mdx [hunk 2]
```diff
@@ -2,21 +2,216 @@
...
+}
+
+const post = await useSanityClient().fetch(`*[_type == "post" && slug.current == slug]`, {slug})
+---
+<article>
+  <h1>{post?.title}</h1>
+  <PortableText value={post?.body} />
+</article>
+```
+If you go beyond these simple examples, then a useful pattern is to collate content queries in [one file and export them as functions][query-file].
```
Comment (ElianCodes) [inline src/content/docs/en/guides/cms/sanity.mdx:183]:
``````suggestion
```

If you go beyond these simple examples, then a useful pattern is to collate content queries in [one file and export them as functions][query-file].
``````

File: src/content/docs/en/guides/cms/sanity.mdx [hunk 3]
```diff
@@ -2,21 +2,216 @@
...
+
+
+
+
+## Official Resources
```
Comment (ElianCodes) [inline src/content/docs/en/guides/cms/sanity.mdx:188]:
```suggestion

## Official Resources
```

File: src/content/docs/en/guides/cms/sanity.mdx [hunk 4]
```diff
@@ -2,21 +2,216 @@
...
 ---
 
-[Sanity](http://sanity.io) is a headless content management system that focuses on [structured content](https://www.sanity.io/structured-content-platform).
 
-## Official Resources
+You can use [Sanity](http://sanity.io) as a headless content management system (CMS) to drive content to Astro websites. It comes with a hosted document and asset store called Content Lake, a fully customizeable editing enviroment called Sanity Studio, and APIs that lets you integrate your content to pretty much anything.
+
+## Quick start
+
+The easiest way to get started is with the official [`@sanity/astro` integration][astro-sanity]. You can add it to an existing project or start with a new one.
```
[the remark this answers is not in the document]
Comment (kmelve) [reply src/content/docs/en/guides/cms/sanity.mdx:15]: So I take it you prefer passive voice? ([active voice is often preferred in technical writing](https://developers.google.com/style/voice))

**Comment by ElianCodes** [comment]
hey @kmelve! There are still some suggestions left, just making sure you're aware!

**Comment by sarah11918** [comment]
Hi @kmelve ! I know this one has been taking a while, and I'm sorry this one got off to such a slow start. Normally, I would be the final person to review and polish the text, formatting etc. to match the style to the rest of docs and other guides. However, I can see there are still some outstanding comments to be addressed, and I usually wait until all the other reviews have been resolved.

Are you able to address what's still outstanding, so that I can go ahead and do my editing review? I know it's a long process already with such a significant contribution, but we'd still be happy to continue with this if you are! :raised_hands:

**Comment by kmelve** [comment]
Hi! We have had some challenges with some updates to our integration that would change the API surface. I just wanted to get them settled before submitting an update to the code examples here! Now it's my time to thank you for your patience. 🙇

**Comment by sarah11918** [comment]
Hi @kmelve , we are happy to have this guide updated, but we are getting very close to merging our entire site over to Starlight and we are trying to wrap up outstanding PRs that have been open for a while so that they don't end up in conflicts and problems as we try to migrated.

Since this guide isn't ready to be added to the site, and you are waiting for more content changes on your end, I'm going to close this PR for now, with the following plan:

- I'm committing all of @ElianCodes ' suggestions because they are mostly to improve the writing of existing content. That way, your branch will have already had some good copy edits. This just gives us a better place to start from when we pick it up again!

- You can either reopen, or resubmit fresh when you are able. This will be the best way for us to be able to only have PRs that we are able to merge quickly in our inbox. If this ends up being before the Starlight migration, then I will put this as top priority to edit and merge the content so it's in there! If not, then you will need to reopen a new PR because too much of the site will have changed for us to use this existing PR, but you can use the edits we've made to get the new copy started.

Thank you very much for your understanding! It doesn't usually go quite like this, but we each had a major delay on our end, and omg Docs Never Stops, so sometimes we have to regroup and restrategize!  🙌

**Pull request closed without merging**